data-raw/standard-format-data-prep/rst_effort_standard_format.Rmd

```diff
@@ -210,16 +210,19 @@ butte_hours_fished <- filter(trap, stream == "butte creek") %>%
          hours_fished = round(difftime(end_datetime, start_datetime, units = "hours"), 2),
          hours_fished = ifelse(hours_fished < 0, 24, hours_fished))
 
+# tried treating post 2015 differnt since it is cAMP data and has start and end dates but caused a lot of huge outliers
```

Comment:
Thanks for including this note. Looking at this effort doc again, I'm still not totally satisfied but that is not a Butte issue so we can leave this as is.

Reply:
Agreed. We can get back to this at some point!